.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,70 @@
+name: Bug report
```

Comment:
this and other issue templates seem identical from upstream. are they needed for the ARM build? if so, why?

Reply:
It's not needed. Deleted.